<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSplitter</class>
 <widget class="QDialog" name="ImageSplitter">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Image Splitter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Select image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FilePathBrowser" name="path">
       <property name="placeholderText">
        <string>Select image...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Image size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ImageSizeLbl">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="redIconLbl">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="QtIcons.qrc">:/QtIconsTextureDialog/red.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ImageArea" name="redImgLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Drag png or click to select file.</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="redSpinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redFillBtn">
           <property name="text">
            <string>Fill</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="greenIconLbl">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="QtIcons.qrc">:/QtIconsTextureDialog/green.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ImageArea" name="greenImgLbl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Drag png or click to select file.</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSpinBox" name="greenSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="greenFillBtn">
             <property name="text">
              <string>Fill</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="greenIconLbl_2">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="QtIcons.qrc">:/QtIconsTextureDialog/blue.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ImageArea" name="blueImgLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Drag png or click to select file.</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="blueSpinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="blueFillBtn">
           <property name="text">
            <string>Fill</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="blackIconLbl">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="QtIcons.qrc">:/QtIconsTextureDialog/alpha.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ImageArea" name="alphaImgLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Drag png or click to select file.</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="alphaSpinBox">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="alphaFillBtn">
           <property name="text">
            <string>Fill</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QPushButton" name="restoreBtn">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QPushButton" name="reload">
         <property name="toolTip">
          <string>Realod image</string>
         </property>
         <property name="text">
          <string>Reload</string>
         </property>
         <property name="icon">
          <iconset resource="QtIcons.qrc">
           <normaloff>:/QtIcons/reloadtextures.png</normaloff>:/QtIcons/reloadtextures.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadSpecular">
         <property name="toolTip">
          <string>Reload Specular Map</string>
         </property>
         <property name="text">
          <string>Reload SM</string>
         </property>
         <property name="icon">
          <iconset resource="QtIcons.qrc">
           <normaloff>:/QtIcons/refresh.png</normaloff>:/QtIcons/refresh.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveChannelsBtn">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save channels</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsBtn">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save As</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageArea</class>
   <extends>QLabel</extends>
   <header>Classes/Qt/ImageSplitterDialog/ImageArea.h</header>
  </customwidget>
  <customwidget>
   <class>FilePathBrowser</class>
   <extends>QLineEdit</extends>
   <header>Classes/Qt/Tools/Widgets/FilePathBrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>redSpinBox</tabstop>
  <tabstop>redFillBtn</tabstop>
  <tabstop>greenSpinBox</tabstop>
  <tabstop>greenFillBtn</tabstop>
  <tabstop>blueSpinBox</tabstop>
  <tabstop>blueFillBtn</tabstop>
  <tabstop>alphaSpinBox</tabstop>
  <tabstop>alphaFillBtn</tabstop>
  <tabstop>restoreBtn</tabstop>
  <tabstop>reload</tabstop>
  <tabstop>reloadSpecular</tabstop>
  <tabstop>saveChannelsBtn</tabstop>
  <tabstop>saveAsBtn</tabstop>
  <tabstop>saveBtn</tabstop>
 </tabstops>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
